eucaconsole.init

```diff
@@ -60,9 +70,16 @@ start() {
     fi    
 
     if [ "x$NGINX_FLAGS" != xNO ]; then
-	echo -n "Starting eucaconsole nginx: "
+        echo -n "Starting eucaconsole nginx: "
         daemon /usr/sbin/nginx -c /etc/eucaconsole/nginx.conf $NGINX_FLAGS
-	[ $? -eq 0 ] && touch /var/lock/subsys/eucaconsole-nginx
+        [ $? -eq 0 ] && touch /var/lock/subsys/eucaconsole-nginx
+        echo
+    fi
+
+    if [ "x$MEMCACHED_FLAGS" != xNO ]; then
+        echo -n "Starting eucaconsole memcached: "
+        daemon --user=$CONSOLEUSER --pidfile ${MEMCACHEDPIDFILE} memcached -d -p $PORT -u $CONSOLEUSER  -m $MEMCACHED_CACHESIZE -c $MEMCACHED_MAXCONN -P ${MEMCACHEDPIDFILE} $MEMCACHED_FLAGS
```

Comment:
I haven't had a chance to test it yet, but won't -P cause memcached to try to write its own PID file?  The init script is already taking care of that.